prog/dftb+/include/common.fypp

```diff
@@ -77,5 +77,24 @@
 #{if rank == 0}##{else}#(${":" + ",:" * (rank - 1)}$)#{endif}#
 #:enddef
 
+
+#! Creates a stub subroutine with given arguments which calls raises an error.
+#!
+#! Args:
+#!     NAME: Name of the stub subroutine to create.
+#!     ARGTYPES: List of subroutine argument types.
+#!
+#:def STUB_SUBROUTINE(NAME, ARGTYPES)
+#:set args = ["arg{}".format(iarg + 1) for iarg in range(len(ARGTYPES))]
+subroutine ${NAME}$(${", ".join(args)}$)
+    use message, only : error
```

Comment:
This doesn't work with the current dependency generation mechanism from cr_makedep as the F90 routine using the STUB macro won't have a use at the top for message

Reply:
You're right, I'll fix it.